Align chart name with openshift-charts (#23)
redhat-ai-dev/ai-lab-helm-charts: charts/ai-software-templates/chatbot/README.md

### Install using Helm

In order to install Chatbot AI Sample Helm chart using helm directly, you can run:

```
export APP_NAMESPACE='the-namespace-your-release-will-be-installed'
export RELEASE_NAME='the-name-of-your-release'

> helm upgrade --install $RELEASE_NAME --namespace $APP_NAMESPACE .

Reviewer: Why export these env vars instead of just using the strings inline in the helm chart command?

Author: The idea behind the env vars is to align this command with the PIPELINES_CONFIGURATION.md doc.

Author: FYI I've reverted this commit. Only because the rest of the changes are more urgent. We can continue the discussion and see what is best!

### Install using Helm

> In order to install Chatbot AI Sample Helm chart using helm directly, you can run:

Reviewer 2: ```suggestion
To install the Chatbot AI Sample Helm chart using Helm directly, you can run:
```

Nit
